src/app/products/products.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="products-container">
+  <nav class="sidebar">
+    <ul class="navbar">
+      {{#each categories.categories}}
+        <li class="nav-item">
+          <a class="nav-item-link item-{{@index}}" href="#">
+            <span>{{name}}</span>
+          </a>
+        </li>
+      {{/each}}
+    </ul>
+  </nav>
+  <nav class="topbar">
+    <select class="navbar selected">
```

Comment:
Why a select inside Nav?

Reply:
Because it is used with the role of navigation. So, I thought should wrap it into nav element tag.